backend/pkg/api/types/notifications.go

```diff
@@ -61,19 +61,20 @@ type NotificationEventValidatorBackOnline struct {
 
 type NotificationValidatorDashboardDetail struct {
```

Comment:
should be acked by @LuccaBitfly

Reply:
We discussed these changes privately, should be fine